inst/WORDLIST

```diff
@@ -7,3 +7,4 @@ draggable
 funder
 pre
 repo
+tagList
```

Comment:
it's a technical term - please wrap it in quotes and remove this exception

Reply:
teal.widgets is now in CRAN, I created new issue to update this on next release.